examples/mobile/UIComponents/components/dialogs/pickers/calendar.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta content="width=device-width, user-scalable=no" name="viewport" />
+	<link href="../../../lib/tau/mobile/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../../css/style.css" rel="stylesheet" />
+	<script data-build-remove="false" src="../../../lib/tau/mobile/js/tau.js">
+	</script>
+</head>
+
+<body>
+	<div class="ui-page" data-role="page">
+		<div class="ui-header" data-position="fixed">
+			<h1>
+				Calendar
+			</h1>
+		</div>
+		<div class="ui-content">
```

Comment:
In the Guide, it is included in the List.
So the calendar width is same with List width.
Would you make the sample with List, please?

Reply:
done.